src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -513,6 +513,35 @@ export class BrowsingContextImpl {
         cdpSessionId: this.#cdpTarget.cdpSessionId,
       });
     });
+
+    this.#cdpTarget.cdpClient.on('Page.javascriptDialogClosed', (params) => {
+      this.#eventManager.registerEvent(
+        {
+          method: BrowsingContext.EventNames.UserPromptClosed,
+          params: {
+            context: this.id,
+            accepted: params.result,
+            // Cast empty string to undefined
+            userText: params.userInput ? params.userInput : undefined,
```

Comment:
```suggestion
            userText: params.userInput ?? undefined,
```

Reply:
This not correct, For the spec we can return `undefined` but CDP always response with at least "" so the best we can do is use `||` instead of ternary operator.

Reply:
Add a link to the spec please